grafana/grafana.spec

```diff
@@ -4,17 +4,22 @@
 %global version 6.2.2
 
 Name:             grafana
-Version:          %{version}
-Release:          3%{?dist}
+Version:          6.2.2
```

Comment:
It matches the product configuration and also matches the sed being used in the scripts to update the value should the version change.

We can certainly improve the way it happens in another PR, for both community and product, however the purpose of this PR is to more closely match the product and reduce the deviation.

Reply:
Another reason is the scripts use the value from this line when a version is not explicitly specified so it currently has to be a value rather than reference to the previous global

Reply:
Got it, thanks!